### DISCUSSION TITLE
Any example/support for onclick for tab

### DISCUSSION THREAD

**Discussion by jaymehta-twilio** [twilio-labs/paste#1250] [Q&A] [answered]
Hi, we have a requirement of fetching some data on a page with multiple tabs. It's not final yet but just wanted to check if there's a way to trigger certain actions on select/click of a tab. 

[The useState example](https://paste.twilio.design/components/tabs/#using-state-hooks) is something I was looking for. But, are there any plans to provide onClick/ onSelect hook ? Please suggest if there's an example with current props. Thanks.

**Comment by nkrantz** [comment]
Hi @jaymehta1986, thanks for pointing this out. We're going to discuss this internally and get back to you. Adding an onClick to your tabs should work, but currently our typing is too strict to allow it. The plan is to change that. Stay tuned!

**Reply by jaymehta-twilio** [reply]
Thanks @nkrantz !

**Comment by TheSisb** [comment, marked as answer]
Hi @jaymehta1986, I just merged the fix: https://github.com/twilio-labs/paste/pull/1260
This error should no longer appear in the next release.
